microsoft-365/enterprise/office-365-network-mac-perf-insights.md

```diff
@@ -25,10 +25,11 @@ Insights are intended to help in designing network perimeters for your office lo
 There are six specific network insights that may be shown for each office location:
```

Comment:
```suggestion
There are eight specific network insights that may be shown for each office location:
```
```suggestion
There are six specific network insights that may be shown for each office location:
```

Reply:
I count 7 network insights in the published doc. I don't think we added any so it should still be seven.